app/scripts/modules/amazon/src/instance/details/instanceDetails.html

```diff
@@ -209,16 +209,17 @@ <h3 class="horizontal middle space-between flex-1" select-on-dbl-click>
         <dd ng-if="instance.permanentIps" ng-repeat="ip in instance.permanentIps">
           <link-with-clipboard url="'http://' + ip + ':' + state.instancePort" text="ip"> </link-with-clipboard>
         </dd>
-        <dt ng-if="instance.publicIpAddress || instance.ipv6Addresses.length">
-          Public IP Address<span ng-if="instance.ipv6Addresses.length > 1">es</span>
-        </dt>
+        <dt ng-if="instance.publicIpAddress">Public IP Address</dt>
         <dd ng-if="instance.publicIpAddress">
           <link-with-clipboard
             url="'http://' + instance.publicIpAddress + ':' + state.instancePort"
             text="instance.publicIpAddress"
           >
           </link-with-clipboard>
         </dd>
+        <dt ng-if="instance.ipv6Addresses.length">
+          IPv6 Address<span ng-if="instance.ipv6Addresses.length > 1">es</span>
+        </dt>
```

Comment:
similar question as my other comment re: structure — could there be IPv4 and IPv6 sections under "Public IP Address(es)"?

Reply:
The feedback I had was to keep IPv6 and IPv4 in separate headings for the users since the concept of public vs private does not exist in v6. We started out with exactly what you said. If you look to the changed code, you can see that everything was listed under "Public IP Addresses".  However, we concluded that it is better to isolate v6 since it is its own entity. 

Also, it is extremely unlikely that an instance will have all 3 types of IP addresses. We may see it during this roll out phase, but the common case will be an instance with 1 Private IPv4 and 1 or more IPv6.

Reply:
👍 👍